client/settings/deposits/index.js

```diff
@@ -160,56 +156,47 @@ const DepositsSchedule = () => {
 		depositRestrictions === 'schedule_restricted'
 	) {
 		return (
-			<Notice
-				status="warning"
-				isDismissible={ false }
-				className="deposits__notice"
-			>
-				<span>
-					{ __(
-						'Deposit scheduling is currently unavailable for your store.',
-						'woocommerce-payments'
-					) }
-				</span>
-				<a
-					aria-label={ __(
-						'Learn more about deposit scheduling.',
+			<InlineNotice status="warning" isDismissible={ false } icon>
+				{ interpolateComponents( {
+					mixedString: __(
+						'Deposit scheduling is currently unavailable for your store. {{learnMoreLink}}Learn more{{/learnMoreLink}}',
```

Comment:
I'm guessing changing the link from `Learn more about deposit scheduling` to `Learn more` was about making it more consistent.

Reply:
Yeh, `Learn more about deposit scheduling` was the aria-label of the `?` icon button and seemed too verbose, given we usually use `Learn more` in these notices and the design in issue #6424 showed `Learn more` as well.